contracts/Vault.vy

```diff
@@ -366,7 +376,7 @@ def _state_hash2(listing: Listing, rental: Rental) -> bytes32:
     return keccak256(
         concat(
             rental.id,
-            convert(rental.owner, bytes32),
+            # owner is not part of state, as self.owner already exists
```

Comment:
the comment or the `owner` in `Rental`? the `owner` is useful to send back to the `Renting` contract so that the `owner` can be included in the events. IMO this can be part of the `Rental` struct and not be part of the contract state

Reply:
I meant to ask if we can remove the comment 😅

Reply:
done